### ISSUE TITLE
Potential Bug: Program closes after switching MFD tabs.

### ISSUE THREAD

**Issue by ilikepizza1275** [andrewdsmith/edxlc#4]
Hello. I recently got my X52 Pro and have been trying to get EDXLC to work fully with another plugin I've been using for Elite. When EDXLC starts, it creates a blank page on the MFD screen on the X52 Pro base. Normally this wouldn't be a problem, however when I switch pages to one created by the other plugin, EDXLC stops and closes. I'm not quite sure if this is a bug or an inherent limitation of only one program allowed to talk to the X52 Pro at any given moment. I hope it's the later because using EDXLC and the other plugin would make for the perfect setup for using the X52 Pro with Elite. I'm happy to provide any additional details you may need to solve this issue. Thanks!

Link to aforementioned plugin: [https://github.com/peterbn/EDx52display](url)

**Comment by andrewdsmith** [comment]
Yes EDXLC is brittle around the MFD handling, mainly because handling it properly added extra complexity couldn't be bothered to spend time on as I didn't think a situation like this would crop up. I'm afraid I'm no longer actively maintaining this project as I've moved onto other things, but I would happily accept pull requests.
